pyaerocom/_lowlevel_helpers.py

```diff
@@ -837,11 +837,11 @@ def dict_to_str(dictionary, indent=0, ignore_null=False):
     return s
 
 
-def str_underline(s, indent=0):
+def str_underline(title: str, indent: int = 0):
     """Create underlined string"""
-    s = indent * " " + f"{s}\n"
-    s += indent * " " + f"{len(s) * '-'}"
-    return s
+    lenght = indent + len(title)
```

Comment:
lenght should probably called length, but the variable name in principle does not matter

Reply:
done